aiorospy/requirements.txt

```diff
@@ -1,2 +1,11 @@
-aiounittest==1.2.1
-janus==0.4.0
+aiounittest==1.3.1        # via -r /home/pbovbel/locus_dev/src/aiorospy/aiorospy/requirements.in
```

Comment:
To confirm, this whole file was generated by the new catkin_virtualenv tooling?

Reply:
yep